test/test_overall_state.py

```diff
@@ -0,0 +1,592 @@
+        }
+        requests.patch(self.endpoint + '/host/' + ls_host['_id'], json=data,
+                       headers=headers_patch, auth=self.auth)
+        response = requests.get(self.endpoint + '/host/' + ls_host['_id'],
+                                params=sort_id, auth=self.auth)
+        ls_host = response.json()
+        # _overall_state_id field is 3 (host unreachable)
+        self.assertEqual(3, ls_host['_overall_state_id'])
+
+        # Update live state for an host
+        # => DOWN SOFT (anyway, do not care about HARD state!)
+        time.sleep(0.1)
+        data = {
+            'ls_state': 'DOWN',
+            'ls_state_id': 1,
+            'ls_state_type': 'SOFT',
+            'ls_acknowledged': False,
+        }
+        headers_patch = {
+            'Content-Type': 'application/json',
+            'If-Match': ls_host['_etag']
+        }
+        requests.patch(self.endpoint + '/host/' + ls_host['_id'], json=data,
+                       headers=headers_patch, auth=self.auth)
+        response = requests.get(self.endpoint + '/host/' + ls_host['_id'],
+                                params=sort_id, auth=self.auth)
+        ls_host = response.json()
+        # _overall_state_id field is 4 (host down)
+        self.assertEqual(4, ls_host['_overall_state_id'])
+
+        # we acknowledge the host
+        time.sleep(0.1)
+        data = {
+            'ls_state': 'DOWN',
+            'ls_state_id': 1,
+            'ls_state_type': 'SOFT',
+            'ls_acknowledged': True,
+        }
+        headers_patch = {
+            'Content-Type': 'application/json',
+            'If-Match': ls_host['_etag']
+        }
+        requests.patch(self.endpoint + '/host/' + ls_host['_id'], json=data,
+                       headers=headers_patch, auth=self.auth)
+        response = requests.get(self.endpoint + '/host/' + ls_host['_id'],
+                                params=sort_id, auth=self.auth)
+        ls_host = response.json()
+        # _overall_state_id field is 1 (host down and ack)
+        self.assertEqual(1, ls_host['_overall_state_id'])
+
+        # we downtime the host
+        time.sleep(0.1)
+        data = {
+            'ls_state': 'DOWN',
+            'ls_state_id': 1,
+            'ls_state_type': 'SOFT',
+            'ls_acknowledged': False,
+            'ls_downtimed': True,
+        }
+        headers_patch = {
+            'Content-Type': 'application/json',
+            'If-Match': ls_host['_etag']
+        }
+        requests.patch(self.endpoint + '/host/' + ls_host['_id'], json=data,
+                       headers=headers_patch, auth=self.auth)
+        response = requests.get(self.endpoint + '/host/' + ls_host['_id'],
+                                params=sort_id, auth=self.auth)
+        ls_host = response.json()
+        # _overall_state_id field is 2 (host down and downtimed)
+        self.assertEqual(2, ls_host['_overall_state_id'])
+
+    def test_update_service(self):
```

Comment:
I have do a problem sone days ago, so can you update this test with 2 services on the host, to be sure it works ;)